docs/Installation/Docker.md

```diff
@@ -0,0 +1,41 @@
+description: Run EthSigner from Docker image
+<!--- END of page meta data -->
+
+# Running EthSigner from Docker Image
+
+A Docker image is provided to run EthSigner in a Docker container. 
+
+## Prerequisites
+
+* [Docker](https://docs.docker.com/install/)   
+
+* MacOS or Linux 
+    
+    !!! important 
+        The Docker image does not run on Windows. 
+
+## Quickstart
+
+Display the EthSigner command line help using the Docker image: 
+
+```bash tab="latest"
+docker run pegasyseng/ethsigner:latest --help
+```
+!!! note
+    `latest` runs the latest cached version. To pull the latest version, use `docker pull pegasyseng/pantheon:latest`. 
+ 
+## Expose Listening Port
+
+Exposing the listening port is required to use the  default listening port (`8545`) or the port specified using 
+[`--http-listen-port`](../Reference/EthSigner-CLI.md#http-listen-port). 
+
+To run EthSigner exposing listening port for access: 
+```bash
+docker run -p <listenPort>:8545 pegasyseng/ethsigner:latest <options>
+```
+
+!!!example
+    To run EthSigner using [file-based signing](../Using-EthSigner/Getting-Started.md) using the default listing port and 8590 as the downstream port:
```

Comment:
Perhaps the values that should be substituted should have angle brackets around them like in the Pantheon docs?
That would also that would make it clearer that pantheonHost is something not needs to modified to be their pantheon host and not just copied and pasted.

Reply:
Fixed